### ISSUE TITLE
Broken Link 

### ISSUE THREAD

**Issue by isunitha98selvan** [pelias/pelias#879]
**Describe the bug**

**Comment by missinglink** [comment]
Both these links seem to be working for me?

![Screenshot_20200728-104711](https://user-images.githubusercontent.com/738069/88641629-c1faee00-d0bf-11ea-8878-a43ed6850988.jpg)

**Comment by isunitha98selvan** [comment]
The link to full installation docs redirects me to https://github.com/pelias/documentation/blob/master/pelias_from_scratch.html , which seems to be broken

**Comment by orangejulius** [comment]
Hi @isunitha98selvan,

I see that the link is broken on [https://pelias.io](https://pelias.io), but NOT on Github ([https://github.com/pelias/pelias#how-can-i-install-my-own-instance-of-pelias](https://github.com/pelias/pelias#how-can-i-install-my-own-instance-of-pelias)). Is that what you're seeing?

That might explain the confusion. Thanks for the report, we're planning on overhauling our website soon anyway, where we'll be able to fix broken things like this, hopefully.

**Comment by isunitha98selvan** [comment]
Yes, I was referring to the website. I should have mentioned it earlier, my bad.
